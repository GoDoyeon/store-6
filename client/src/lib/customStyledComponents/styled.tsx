import constructWithTag from '.';
import domElements from './utils/domElements';

interface Styled {
  [tag: string]: (string: TemplateStringsArray, ...args: unknown[]) => Function;
}

const styled: Styled = {};

domElements.forEach((domElement) => {
  styled[domElement] = constructWithTag(domElement);
});

export default styled;
