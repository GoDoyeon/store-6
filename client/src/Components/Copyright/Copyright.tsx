import styled from 'styled-components';

const Footer = styled.footer`
  margin-top: 28px;
  /* font-family: Noto Sans !important; */
  font-style: normal;
  font-weight: normal;
  font-size: 12px;
  line-height: 16px;
  color: #999999;
`;

export default Footer;
