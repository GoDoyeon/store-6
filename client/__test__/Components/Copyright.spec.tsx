import React from 'react';
import styled from 'styled-components';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import Copyright from '~/Components/Copyright';

describe('<Button />', () => {
  it('should render same with snapshot', () => {
    const { container } = render(<Copyright>저작권 관련 내용</Copyright>);
    expect(container).toMatchSnapshot();
  });
});
