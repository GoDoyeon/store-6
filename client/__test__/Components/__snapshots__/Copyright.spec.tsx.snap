// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render same with snapshot 1`] = `
.c0 {
  margin-top: 28px;
  font-style: normal;
  font-weight: normal;
  font-size: 12px;
  line-height: 16px;
  color: #999999;
}

<div>
  <footer
    class="c0"
  >
    저작권 관련 내용
  </footer>
</div>
`;
